layers/+distributions/spacemacs-bootstrap/packages.el

```diff
@@ -229,9 +229,13 @@
   (spacemacs|define-transient-state paste
     :title "Pasting Transient State"
     :doc "\n[%s(length kill-ring-yank-pointer)/%s(length kill-ring)] \
- [_C-j_/_C-k_] cycles through yanked text, [_p_/_P_] pastes the same text \
- above or below. Anything else exits."
+ [_C-j_/_C-k_] cycles through yanked text, [_p_/_P_] pastes the \
+ same text above or below, [_C-v_] creates a visual selection \
+ from last paste and exits. Anything else exits."
     :bindings
+    ("C-v" (lambda (x) (interactive "*p")
```

Comment:
why is "*p" necessary here?

the lambda form never uses the extra numerical prefix argument (from "p").

Reply:
Indeed, I will remove the unused argument, a remnant from a more complicated version before I realized that (evil-active-region) does the job.